app/src/main/java/com/eis/audioUtility/AudioUtilityManager.java

```diff
@@ -0,0 +1,199 @@
+package com.eis.audioUtility;
+
+/**
+ * This is an Utility class used to manage main Audio Streams (Alarm, Music, and Ring),
+ * and to manage Vibration
+ *
+ * @author Francesco Bau'
+ */
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+import static android.content.Context.AUDIO_SERVICE;
+
+
+public class AudioUtilityManager {
+
+    public static final int MAX_PERCENTAGE = 100;
+    public static final int MIN_PERCENTAGE = 0;
+
+    private static final int VIBRATION_TIME = 500;
+    private static final int VIBRATION_DELAY = 500;
+    private static final int VIBRATION_REPEAT_CODE = 1;
+
+    public enum AUMStream {
+        ALARM,
+        RING,
+        MUSIC
+    }
+
+    /**
+     * @param context The current Context.
+     * @return The current AudioManager instance.
+     */
+    protected static AudioManager getAudioManager(Context context) {
+        return (AudioManager) context.getSystemService(AUDIO_SERVICE);
+    }
+
+    /**
+     * @param context The current Context.
+     * @param stream  The chosen stream (should be ALARM, RING or MUSIC).
+     * @return The current stream Volume (in percentage).
+     */
+    public static int getVolume(Context context, AUMStream stream) {
+        int currentVolume = getAudioManager(context).getStreamVolume(getStream(stream));
+
+        int maxVolume = getMaxVolume(context, stream);
+        return Math.round(MAX_PERCENTAGE * currentVolume / maxVolume);
+    }
+
+    /**
+     * @param context The current Context.
+     * @param stream  The chosen stream (should be ALARM, RING or MUSIC).
+     * @return The maximum stream volume (real value).
```

Comment:
Specify that this value is not always a percentage

Reply:
In fact, it's not a percentage, it's the real value.
So should I write a better description of this method?
This one is important, because it's needed to calculate the percentage...

Reply:
You should say in the @return that the maximum stream volume is not a percentage

Reply:
ok, thank you for the advice.